resources/js/Pages/Praktikan.vue

```diff
@@ -1674,13 +1672,10 @@ export default {
             // Start opening Jurnal Section and get all SOAL from
             // get soal from soal__jurnals and soal__fitbs 
             // new: get jawaban from jawaban__tp where soal__tp.isProgram=1
-            globe.tpPickRandomProgram();
+            randomNumber = globe.tpPickRandomProgram();
             globe.$axios.post('api/getTp/'+globe.currentUser.nim+'/'+globe.current_praktikum.modul_id).then(response => {
-            globe.soalPresentasi = JSON.stringify(response.data.all_tp[globe.randomNumber].jawaban)
-             console.log(JSON.stringify(response.data.all_tp[globe.randomNumber].jawaban))
-             console.log(globe.currentUser.nim)
-             console.log(globe.current_praktikum.modul_id)
-             console.log(globe.soalPresentasi)
+            globe.soalPresentasi = JSON.stringify(response.data.all_tp[randomNumber].jawaban)
+            
               // if(response.data.message === "success") {
 
               //   if(response.data.all_soal !== null){
```

Comment:
remove all this commented lines too, its simply unnecessary to save all commented lines when using git

Reply:
remove all commented lines in **every files**

Reply:
all commented **script** or literally **all comments** ?

Reply:
the code, and yeah sure comments are still okay, but commented code are just a trash basically